python/tests/conftest.py

```diff
@@ -1,3 +1,15 @@
+import pytest
 import logging
 
+import seldon_core
+
+
 logging.basicConfig(level=logging.DEBUG)
+
+
+@pytest.fixture(params=[True, False])
+def client_gets_metrics(monkeypatch, request):
+    value = request.param
+    monkeypatch.setattr(seldon_core.user_model, "CLIENT_GETS_METRICS", value)
```

Comment:
Would it work if you patch the env var directly with `monkeypatch.setenv()`?

Reply:
I don't think so because environmental variable is read when importing the module.